hbase_sql/src/test/java/com/splicemachine/derby/utils/VacuumIT.java

```diff
@@ -105,46 +100,77 @@
             .around(spliceTableIWatcher)
             .around(spliceTableJWatcher);
 
+    @Before
+    public void waitForStaleTransactions() throws Exception
```

Comment:
Can you create a JIRA to remove this and reference it here?

Reply:
https://splicemachine.atlassian.net/browse/DB-10490

Reply:
I meant referencing the JIRA in a comment as a TODO for instance

Reply:
done